ConnectorDynamo/ConnectorDynamo.slnf

```diff
@@ -23,7 +23,7 @@
       "Objects\\Converters\\ConverterRevit\\ConverterRevit2024\\ConverterRevit2024.csproj",
       "Objects\\Converters\\ConverterRevit\\ConverterRevitShared\\ConverterRevitShared.shproj",
       "Objects\\Objects\\Objects.csproj",
-      "Objects\\Tests\\Objects.Tests.Unit\\Objects.Tests.Unit.csproj"
```

Comment:
Damn, I may have made this mistake elsewhere. I'll double check and PR if necessary.

Reply:
Just double checked, the `Objects.Tests.Unit.csproj` is the last item in every other slnf except this one. so only this one was broken. Thanks for fixing.